index.bs

```diff
@@ -2554,12 +2554,45 @@ as it implies that it's possible to dispatch an event asynchronously.
 All events are dispatched synchronously.
 What is more often implied by "asynchronous event" is to defer firing an event.
 
-<h3 id="state-and-subclassing">Use plain {{Event}}s for state</h3>
+<h3 id="state-and-subclassing">Put state on {{Event/target}} objects rather than {{Event}}s</h3>
 
-Where possible, use a plain {{Event}} with a specified `type`,
-and capture any state information in the {{Event/target}} object.
+Where possible, rather than put state on {{Event}} objects, put state on the {{Event/target}}
+and use events to signal updates to that state.
```

Comment:
```suggestion
Put state on the {{Event/target}}
and use events to signal updates to that state,
rather than having state on {{Event}} objects.
```